### DISCUSSION TITLE
Between Operator for Eloquent

### DISCUSSION THREAD

**Discussion by njames** [laravel/framework#35639] [General]
It would be nice to have a Between operator in Eloquent

as in 
```
    User::whereBetween('created_at', ['2020-12-24', '2020-12-26'])
```

we use dates in queries quite a lot and this would be helpful for us. 

I am prepared to have a crack a pr but thought the discussion would be useful first to see if others think this is useful. 

Also `whereNotBetween`

It obviously would work with types other than dates, but dates are my itch. 

Cheers, 
Nigel
[@njames](https://twitter.com/njames/)

**Comment by ManojKiranA** [comment]
I think Query builder alreay has those methods check out [here](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Database/Query/Builder.php#L1118-L1146)

**Reply by njames** [reply]
Huh ... thanks ... I was diving in the grammar files the other day and couldn't see those.
Job done.
